modules/DMZ/data_kit/data_analysis.py

```diff
@@ -0,0 +1,24 @@
+# This is for analyzing columns or datasets, to figure out how to best act upon them.
+
+
+# This will take the target "predicted" column and decide if classification or regression should be used.
+def get_prediction_type(target_column):
+    sorted_data = sorted(target_column)
+
+    prediction_type = "classification"
+
+    last = None
```

Comment:
Last is a Boolean? should this be false or None.  Does this have the same effect.  If this is no issue, just ignore it.